lib/LedgerSMB/Report.pm

```diff
@@ -281,8 +281,8 @@ sub _render {
                        $srt_b = $srt_b->to_sort
                            if eval { $srt_b->can('to_sort') };
                    }
-                   no warnings 'numeric';
-                   $srt_a <=> $srt_b or $srt_a cmp $srt_b;
+
+                   $srt_a cmp $srt_b;
```

Comment:
Yes, let me know.   My thinking:
To make sense it needs `/[-\.\+]{0,1}\d+.*/` or /\D+/ values, but not both.  But a user could have a convention where positive ints are normal, and specials are non-numeric and so can be easily segregated.  But I did not get so far as seeing if such a convention would make any sense in with the sorted objects.  Or whether the "missorting" would be a significant aggravation to a user.
The list given above is an actual sorted sample.